src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -2379,6 +2379,111 @@ paths:
                 $ref: '#/components/schemas/ErrorModel'
       x-codegen-request-body-name: upgrade
 
+##############################################################################
+## SEARCH API STANDARD ENDPOINTS
+##############################################################################
+  /api/repository/v1/search/{id}/index:
+    post:
+      tags:
+        - search
+        - snapshots
+        - repository
+      description: Create a new search index for a snapshot
```

Comment:
This is good, in the future we may want to pass in a type argument if we support indexing different types of objects.